test/integration/master/audit_dynamic_test.go

```diff
@@ -274,7 +274,7 @@ func asyncOps(
 				continue
 			}
 			e := expected.Load().([]utils.AuditEvent)
-			evList := []utils.AuditEvent{}
+			var evList []utils.AuditEvent
 			evList = append(e, exp...)
```

Comment:
fix the hint `this value of evList is never used`?
seem just change the define

Reply:
it seems better.thanks